pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -727,6 +727,8 @@ private void writeMetadata()
           if (_totalDocs > 0) {
             String startTimeStr = timeColumnIndexCreationInfo.getMin().toString();
             String endTimeStr = timeColumnIndexCreationInfo.getMax().toString();
+            properties.setProperty(RAW_SEGMENT_START_TIME, startTimeStr);
```

Comment:
We don't need to persist them as new fields because they are just the min/max value for the time column.